### PR TITLE
Update dev-server.md

### PR DESCRIPTION
**Pull request by tigt**
* Remove “simple” as per the style guide
* Avoid “whitelist” for racial connotations
* Stronger link text for accessibility

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -43,30 +43,29 @@ module.exports = {
...
+When the server starts, there will be a message prior to the list of resolved modules:
 
 ```bash
 http://localhost:9000/
 webpack output is served from /build/
 Content not from webpack is served from /path/to/dist/
 ```
 
-that will give some background on where the server is located and what it's serving.
+…that will give some background on the server's address and what it's serving.
```
Comment (rishabh3112) [inline src/content/configuration/dev-server.md]:
```suggestion
that will give some background on the server's address and what it's serving.
```

File: src/content/configuration/dev-server.md [hunk 2]
```diff
@@ -742,12 +722,11 @@ module.exports = {
 };
 ```
 
-
 ## `devServer.noInfo` 🔑
 
 `boolean`
 
-Tells dev-server to supress messages like the webpack bundle information. Errors and warnings will still be shown. `devServer.noInfo` is disabled by default.
+Suppresses messages like the webpack bundle information. Errors and warnings will still be shown. `devServer.noInfo` is disabled by default.
```
Comment (rishabh3112) [inline src/content/configuration/dev-server.md]: the previous version is better

Comment (tigt) [reply src/content/configuration/dev-server.md]: Alright. I’ll still update it to spell “suppress” correctly, though

File: src/content/configuration/dev-server.md [hunk 3]
```diff
@@ -43,30 +43,29 @@ module.exports = {
...
 http://localhost:9000/
 webpack output is served from /build/
 Content not from webpack is served from /path/to/dist/
 ```
 
-that will give some background on where the server is located and what it's serving.
+That will give some background on the server's address and what it's serving.
 
-If you're using dev-server through the Node.js API, the options in `devServer` will be ignored. Pass the options as a second parameter instead: `new WebpackDevServer(compiler, {...})`. [See here](https://github.com/webpack/webpack-dev-server/tree/master/examples/api/simple) for an example of how to use webpack-dev-server through the Node.js API.
+If you're using dev-server through [the Node.js API](https://webpack.js.org/api/node/), the options in `devServer` will be ignored. Pass the options as a second parameter instead: `new WebpackDevServer(compiler, {...})`. [See an example of how to use webpack-dev-server with the Node.js API.](https://github.com/webpack/webpack-dev-server/tree/master/examples/api/simple)
```
Comment (montogeek) [inline src/content/configuration/dev-server.md]:
```suggestion
If you're using dev-server through [the Node.js API](/api/node/), the options in `devServer` will be ignored. Pass the options as a second parameter instead: `new WebpackDevServer(compiler, {...})`. [See an example of how to use webpack-dev-server with the Node.js API.](https://github.com/webpack/webpack-dev-server/tree/master/examples/api/simple)
```

**Review by montogeek — DISMISSED** [review]
Thanks, really helpful!

**Comment by rishabh3112** [comment]
@wizardofhogwarts there are some merge conflicts that must be resolved first to run CI.

**Comment by chenxsan** [comment]
Closed as it's stale for a long time and the `dev-server` has been changed a lot now.

**Pull request closed without merging**